up42/catalog.py

```diff
@@ -6,14 +6,14 @@
 from pathlib import Path
 from typing import Any, Dict, List, Optional, Union
 
-from geojson import Feature, FeatureCollection
-from geopandas import GeoDataFrame
-from shapely.geometry import Polygon
+from geojson import Feature, FeatureCollection  # type: ignore
+from geopandas import GeoDataFrame  # type: ignore
+from shapely.geometry import Polygon  # type: ignore
 from tqdm import tqdm
 
```

Comment:
I don't understand the reason to introduce these changes since when I run mypy check on the master version it raises only 3 errors related to implicit optional

Reply:
If I do not add these, I see these 3 errors (similar to what you are getting I guess)?
Do you have any alternative solution?

<img width="977" alt="image" src="https://github.com/up42/up42-py/assets/54374009/60278c91-db58-4cc2-84be-8e5c83be403e">

Reply:
I didn't mean imported module, I meant the changes in the code

Reply:
I could not reproduce.
Reverted the un-necessary changes.